src/Xamarin.Android.Build.Tasks/UnitTests/Xamarin.ProjectTools/Xamarin.ProjectTools.csproj

```diff
@@ -0,0 +1,115 @@
+<?xml version="1.0" encoding="utf-8"?>
+<Project DefaultTargets="Build" ToolsVersion="4.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+	<Import Project="..\..\..\..\Configuration.props" />
+	<PropertyGroup>
+		<Configuration Condition=" '$(Configuration)' == '' ">Debug</Configuration>
+		<Platform Condition=" '$(Platform)' == '' ">AnyCPU</Platform>
+		<ProjectGuid>{2DD1EE75-6D8D-4653-A800-0A24367F7F38}</ProjectGuid>
+		<OutputType>Library</OutputType>
+		<RootNamespace>Xamarin.ProjectTools</RootNamespace>
+		<AssemblyName>Xamarin.ProjectTools</AssemblyName>
+		<TargetFrameworkVersion>v4.5</TargetFrameworkVersion>
+	</PropertyGroup>
+	<PropertyGroup Condition=" '$(Configuration)|$(Platform)' == 'Debug|AnyCPU' ">
+		<DebugSymbols>true</DebugSymbols>
+		<DebugType>full</DebugType>
+		<Optimize>false</Optimize>
+		<OutputPath>bin\Debug</OutputPath>
```

Comment:
We should consider putting this into `$(topdir)/bin/Test$(Configuration)`, i.e. `..\..\..\bin\TestDebug`. This is what the Java.Interop repo does.